### PR TITLE
New perltidy is broken

### PR DESCRIPTION
**Pull request by mudler**
New perltidy it spurts an empty file, thus all builds are failing.

Just to see if downgrading fix the issue - cherry picked a failing PR

### CODE REVIEW

**Comment by mudler** [comment]
works here: https://travis-ci.org/os-autoinst/os-autoinst-distri-opensuse/builds/343793835

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ addons:
 before_install:
   - eval $(curl https://travis-perl.github.io/init) --perl
   - echo "requires 'Code::DRY';" >> cpanfile
+  - echo "requires 'Perl::Tidy', '< 20180219';" >> cpanfile
```
Comment (foursixnine) [inline .travis.yml]: I would lock it to 20180219

Comment (mudler) [reply .travis.yml]: but that's the one failing

Comment (mudler) [reply .travis.yml]: we could use 'conflicts ..' and specify the broken version too, let's try

Comment (foursixnine) [reply .travis.yml]: LOL sorry, i meant 20180101

**Comment by mudler** [comment]
Travis is slow picking up the build, on my fork it passed : 
https://travis-ci.org/mudler/os-autoinst-distri-opensuse/builds/343814108

**Pull request merged**